sql/char_equip_saved.sql

```diff
@@ -0,0 +1,26 @@
+SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
+-- ----------------------------
+-- Table structure for char_equip_saved
+-- ----------------------------
+DROP TABLE IF EXISTS `char_equip_saved`;
```

Comment:
This should probably be a protected table in dbtool, and not sure if we want to have the drop here (would suggest following whatever is used in other protected tables)

Reply:
Added to dbtool.py as a protected table.